packages/scripts/README.md

```diff
@@ -435,13 +436,38 @@ This is how you execute those scripts using the presented setup:
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
+* `npm run test:unit:debug` - runs all unit tests with the node debugger enabled.
 
 Jest will look for test files with any of the following popular naming conventions:
 
 - Files with `.js` (or `.ts`) suffix located at any level of depth in `__tests__` folders.
 - Files with `.js` (or `.ts`) suffix directly located in `test` folders.
 - Files with `.test.js` (or `.test.ts`) suffix.
 
+#### Debugging Jest unit tests
```

Comment:
It feels to be highly related to e2e tests, as well. I'm wondering what other scripts would benefit from node debugging support.

Reply:
I think e2e tests are going to be a little weird with half of the execution context happening in the browser. You'd have to run `PUPPETEER_HEADLESS=false` to debug the browser context and have a separate debugging tool open to debug the node context. I found a [puppeteer issue](https://github.com/puppeteer/puppeteer/issues/398#issuecomment-323551586) talking about it. I'll see if I can get the e2e tests running locally and add a note about debugging them if I get it to work.

Reply:
I don’t think you necessary need to run an actual browser to debug the test code itself, not the app controlled by Puppeteer. I guess it’s less useful than in the case of unit tests where you use source of the code rather than what packages and webpack produce and there is no Dev Tools involved.

Reply:
While you can set a breakpoint _almost_ anywhere within the tests, it doesn't work within a `page.evaluate` call because that function is actually running in the browser context.

Since there were a few code changes to get it to work, I opened up #21845 for e2e debugging, and I'll add the docs there.

Reply:
> While you can set a breakpoint _almost_ anywhere within the tests, it doesn't work within a `page.evaluate` call because that function is actually running in the browser context.

Yes, I would expect that to happen 👍 

> Since there were a few code changes to get it to work, I opened up #21845 for e2e debugging, and I'll add the docs there.

Thank you, scheduled for review next week :)